frontend/src/components/PictureOfTheDay.tsx

```diff
@@ -9,7 +9,7 @@ export default function PictureOfTheDay (){
     }
     return <div className={"picture-of-the-day"}>
         <p>{pictureOfTheDay.title}</p>
-        <img src={pictureOfTheDay.url}/>
+        <img src={pictureOfTheDay.url} alt={""}/>
```

Comment:
Why this "alt" is Empty?

Reply:
according to the test that didn't work an alt can have an empty string